### DISCUSSION TITLE
code server authenticate with github

### DISCUSSION THREAD

**Discussion by cristianogregnanin** [coder/code-server#4818] [General]
I go in my code server typing https://ip:8443 then i click bottom-left to configure my github account but nothing happens!
I want to login to sync my extensions.
I have installed code-server on docker

**Comment by lg-nineoneninethreenine** [comment]
Hmm... That's unusual. Can you share a video recording as well as `code-server --log debug`. I hope that command is right🤞

**Reply by jsjoeio** [reply]
Settings Sync doesn't work on code-server I believe :( The official one at least. There's one that uses a gist though that might work.
